### PR TITLE
Enhance existing event-notification-nodejs-sdk to support new topic:PRIORITY_LISTING_REVISION

### PR DESCRIPTION
**Pull request by skumaravelayutham**
Enhance existing event-notification-nodejs-sdk to support new topic:PRIORITY_LISTING_REVISION

### CODE REVIEW

File: lib/processor/priorityListingRevisionMessageProcessor.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/*
+ * *
+ *  * Copyright 2021 eBay Inc.
```
Comment (LokeshRishi) [inline lib/processor/priorityListingRevisionMessageProcessor.js]: Please update this to the current year

**Comment by LokeshRishi** [comment]
@skumaravelayutham Let's uprev the package version as well?

File: lib/processor/priorityListingRevisionMessageProcessor.js (hunk 1 above)
Comment (skumaravelayutham) [reply lib/processor/priorityListingRevisionMessageProcessor.js]: done

Comment (LokeshRishi) [reply lib/processor/priorityListingRevisionMessageProcessor.js]: Looks like `lib/processor/accountDeletionMessageProcessor.js` was updated instead?

Comment (skumaravelayutham) [reply lib/processor/priorityListingRevisionMessageProcessor.js]: updated the right one, reverted the old change

**Review by LokeshRishi — APPROVED** [review]

**Pull request merged**